save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/Organization.kt

```diff
@@ -0,0 +1,16 @@
+package org.cqfn.save.entities
+
+import java.time.LocalDateTime
+import javax.persistence.Entity
+
+/**
+ * @property name organization
+ * @property owner organization
+ * @property dateCreated date created organization
+ */
+@Entity
+class Organization(
```

Comment:
`project` should have mapping to organization too, but it can be done simply with `organization_id` field

Reply:
Yes, added. But I will take out the refactoring of the code and the change in logic in a separate PR.